Easydict/Feature/Utility/EZCoordinateUtils/EZCoordinateUtils.m

```diff
@@ -30,7 +31,11 @@ + (CGPoint)getSafeLocation:(CGRect)frame inScreen:(NSScreen *)screen {
 /// Make sure frame show in screen visible frame, return left-bottom postion frame.
 + (CGRect)getSafeAreaFrame:(CGRect)frame inScreen:(nullable NSScreen *)screen {
     if (!screen) {
-        screen = [self screenOfMousePosition];
+        if (Configuration.shared.showWindowMultiScreen == EZShowWindowMultiScreenAuto) {
+            screen = [self screenOfMousePosition];
+        } else {
+            screen = NSScreen.mainScreen; // EZShowWindowMultiScreenFixed fixed in main screen
```

Comment:
No, it should not use mainScreen here, it should use the screen when selecting words.